examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -576,19 +576,24 @@ def main():
                 pipeline.unet.set_default_attn_processor(pipeline.unet)
 
                 if args.unet_adapter_name_or_path is not None:
-                    from peft import PeftModel
+                    from peft import PeftModel, tuners
+
+                    tuners.boft.layer._FBD_CUDA = False
 
                     pipeline.unet = PeftModel.from_pretrained(pipeline.unet, args.unet_adapter_name_or_path)
-                    pipeline.unet = pipeline.unet.merge_and_unload()
+                    with torch.autocast(device_type="hpu", dtype=torch.bfloat16, enabled=args.bf16):
+                        pipeline.unet = pipeline.unet.merge_and_unload()
```

Comment:
I'm curious, autocast has an impact on merge_and_unload ?

Reply:
has impact on oft in lazy model, remove the autocast will lead to "RuntimeError: Graph compile failed. synStatus=synStatus 26 [Generic failure]"